### DISCUSSION TITLE
Is there a way to check if a specific body is sleeping?

### DISCUSSION THREAD

**Discussion by bovacu** [amerkoleci/JoltPhysicsSharp#39] [Q&A] [answered]
As the titel says, I'd like to know if there is a way to check if a body is sleeping, I have not been able to find how to do it, nor I have found an example on the samples application.

Thanks.

**Comment by amerkoleci** [comment, marked as answer]
Hi,
IsActive should check if body is sleeping https://github.com/jrouwe/JoltPhysics/blob/master/Jolt/Physics/Body/Body.h#L58, so the Api is there, did I miss something?

**Reply by bovacu** [reply]
Oh man, I completely missread it, sorry for the bother...
